# keyframe info sent from the client to the server
float64 time_stamp
uint64 id
uint64 kf_id
std_msgs/Float64MultiArray tcw_opti
FeatureData[] features
